Makefile

```diff
@@ -4,15 +4,29 @@ SHELL := /bin/bash
 ### Without istio
 
 install-local:
-	kubectl apply releases/ -f .
+	kubectl apply -f releases/
+	install-kafka-strimzi
 
 delete:
 	kubectl delete -f releases/
+	delete-kafka-strimzi
 
 ### With Istio
 
 install-local-istio:
 	kubectl apply releases-istio/ -f .
+	install-kafka-strimzi
 
 delete-istio:
-	kubectl delete -f releases-istio/
\ No newline at end of file
+	kubectl delete -f releases-istio/
+	delete-kafka-strimzi
+
+### Kafka Strimzi
+
+install-kafka-strimzi:
+	helm repo add strimzi https://strimzi.io/charts/
+	helm install 0.16.0-rc1 strimzi/strimzi-kafka-operator
```

Comment:
Move this to `https://github.com/stakater-lab/nordmart-dev-tools/` and we follow infrastructure as code everywhere so create a helm chart for this(operator).

Reply:
And once you do that please remove `install-kafka-strimzi` and `delete-kafka-strimzi` from here.

Reply:
Done. Please check this PR https://github.com/stakater-lab/nordmart-dev-tools/pull/18